src/application/Application.mjs

```diff
@@ -103,11 +103,22 @@ export default class Application extends Component {
     }
 
     __updateFocus() {
+        const prevFocusedComponent = this._focusPath ? this._focusPath[this._focusPath.length - 1] : undefined;
         const notOverridden = this.__updateFocusRec();
 
         if (!Application.booting && notOverridden) {
             this.updateFocusSettings();
         }
+
+        if (this.__options.debug) {
+            const newFocusedComponent = this._focusPath ? this._focusPath[this._focusPath.length - 1] : undefined;
+            prevFocusedComponent && prevFocusedComponent.updateDebugFocusState && prevFocusedComponent.updateDebugFocusState();
```

Comment:
Rather cumbersome safety here, but we're only doing this if we're in debug mode. Is there an idiomatic way of handling this that would look nicer and/or be more performant?

Reply:
I've added a comment in code to draw the eye of eg the Lightning guys eventually. But, having looked again I don't think there's a cleaner way to do it without polluting the 'non-inspected' version of Elements.